README.md

```diff
@@ -1,22 +1,24 @@
-# CTF write-ups
-The goal of this repository is to keep track of our participations in online and physical CTFs. By doing a write-up, you will help security newcomers to get engage in our community.
+# CTF Write-ups
+The goal of this repository is to keep track of our participations in online and physical CTFs. By doing a write-up, you will help security newcomers to get engaged in our community.
 
 ## Guidelines
 - Contribute by submiting PRs.
-- Don't be shy to present your solutions.
+- Consider wording your write-up in a way that is accessible to all your fellow JDIs.
+- Don't be shy to present your solutions!
 - If you find a mistake, consider submiting a PR to fix it.
-- If you are doing a live CTF, consider waiting the end before submiting your PRs.
+- If you are doing a live CTF, consider waiting for the end before submiting your PRs (take notes though!).
 - Each event/CTF should have its own folder.
-- Add a README.md in every new folder to give at least a minimal description of the event.
+- Please add a README.md in every new folder to give at least a minimal description of the event.
 
 ## Folder hierarchy
 ```
 write-ups
-|-- CTF name
+|-- CTF_name
 |     |-- README.md (event description)
-|     +-- pwn 42
-|          |-- README.md (your solution)
-|          +-- Extra files
+|     +-- pwn_42
+|     |    |-- README.md (your solution)
+|     |    +-- Extra files
+      +-- simple_challenge.md (if it does not require other files)
```

Comment:
It's not meant to be a folder though. Not sure what the formatting convention is being used here.